### PR TITLE
Project page update

### PR DESCRIPTION
**Pull request by venkanavaneetham**
KC, updated projects page based on new UI and release noted, thanks.

### CODE REVIEW

File: User Guide/Projects.md [hunk 1]
```diff
@@ -121,40 +129,49 @@ After creating a new project, go to **Connections** tab. This tab shows the **co
 - SSL Client certificate
 - SSL Private key 
 - Private key password - you will receive this in your email.
+- Click Download certificate, pop-up to download the certs in .zip, enter a password to save the file in your local.
```
Comment (lohkianchai) [inline User Guide/Projects.md]: "save the file in your local." -> "save the zip file in your computer."

File: User Guide/Projects.md [hunk 2]
```diff
@@ -121,40 +129,49 @@ After creating a new project, go to **Connections** tab. This tab shows the **co
...
+- Alternatively you can also copy and save the certificates in a .PEM text editor file.
```
Comment (lohkianchai) [inline User Guide/Projects.md]: "save the certificates in a .PEM text editor file." -> "save the certificates as a .pem files using a text editor."

File: User Guide/Projects.md [hunk 3]
```diff
@@ -121,40 +129,49 @@ After creating a new project, go to **Connections** tab. This tab shows the **co
...
+**Note:**
+ *New **SSL certificate expires** after 2 years instead of 1 year*.
+ 
 ![Image not Available](/assets/Fig36.png)
 
- **Note:** *The **Private key password** for the projects, you will receive in your email. You must copy and save the certificates in a .PEM text editor file.*
+![Image not Available](/assets/Fig105_2.png)
+
+ **Note:** 
+ *The **Private key password** for the projects, you will receive in their email inbox*.
```
Comment (lohkianchai) [inline User Guide/Projects.md]: "the projects, you will receive in their email inbox" -> "the project will be sent to your email"

File: User Guide/Projects.md [hunk 4]
```diff
@@ -121,40 +129,49 @@ After creating a new project, go to **Connections** tab. This tab shows the **co
...
+![Image not Available](/assets/Fig105_2.png)
+
+ **Note:** 
+ *The **Private key password** for the projects, you will receive in their email inbox*. 
  
  ![Image not Available](/assets/Fig99f.png)
 
- - Alternatively, click **Generate certificate** button to generate new certificate if your **project certificate is expired!**
-
+ - Alternatively, click **Generate certificate** button to generate new certificate if their **project certificate is expired!**
```
Comment (lohkianchai) [inline User Guide/Projects.md]: "their" -> "the"

File: User Guide/Projects.md [hunk 5]
```diff
@@ -121,40 +129,49 @@ After creating a new project, go to **Connections** tab. This tab shows the **co
...
+
```
Comment (lohkianchai) [inline User Guide/Projects.md]: Trim all trailing whitespaces.

File: User Guide/Projects.md [hunk 6]
```diff
@@ -121,40 +129,49 @@ After creating a new project, go to **Connections** tab. This tab shows the **co
...
+ 
  You will receive a **Private key password** through email and use that password in the **Client key passphrase** for your MQTT client set up.
 
 ![Image not Available](/assets/Fig38.png)
 
 
 To set up **MQTT Client connection**:
 
-**Step 1:**	Copy the certificates in a text Editor and save each as .PEM file separately.
+**Step 1:** Click Download certificate, you can see a pop-up to download SSL certs as .pem file in .zip file, enter a password to save the file in your local.
```
Comment (lohkianchai) [inline User Guide/Projects.md]: "save the file in your local." -> "save the file to your computer."

File: User Guide/Projects.md [hunk 7]
```diff
@@ -222,40 +238,41 @@ After creating a new project, go to **Connections** tab. This tab shows the **co
...
 
-If user didn't save the certs file for the first time, user can regenerate the certs by clicking **Generate new certificate** button.
+If you didn't save the SSL certs files for the first time, you can regenerate the certs by clicking **Generate new certificate** button, you can select an **extended grace period** for SSL client certificate.
+
+![Image not Available](/assets/Fig105_3.png)
 
 ![Image not Available](/assets/Fig99f.png)
 
-User will receive a **Private key password** through email and use that password in the **Client key passphrase** for your MQTT client set up.
+You will receive a **Private key password** through email and use that password in the **Client key passphrase** for their MQTT client set up.
```
Comment (lohkianchai) [inline User Guide/Projects.md]: "their" -> "the"

File: User Guide/Projects.md [hunk 8]
```diff
@@ -222,40 +238,41 @@ After creating a new project, go to **Connections** tab. This tab shows the **co
...
 
 ![Image not Available](/assets/Fig38.png)
 
- User will receive the email for **project cert expires soon**!
+You will receive the email for **project cert expires soon**!
 
- User will receive the **Project cert expires soon!** email, 30 days before the 3 certs expires!
+You will receive the **Project cert expires soon!** email, 30 days before the 3 certs expires!
```
Comment (lohkianchai) [inline User Guide/Projects.md]: "certs expires!" -> "certs expire!"

File: User Guide/Projects.md [hunk 9]
```diff
@@ -222,40 +238,41 @@ After creating a new project, go to **Connections** tab. This tab shows the **co
...
 
-
  To set up **MQTT Client connection**:
  
  How to **set up MQTT client connection**:
  
  ![Image not Available](/assets/vidsetupmqttcon.gif)
 
-**Step 1:**	Copy the certificates in a text Editor and save each as .PEM file separately.
+**Step 1:** Download the SSL certificates in a .zip file and save, you can unzip and can see 3 .PEM files separately.
```
Comment (lohkianchai) [inline User Guide/Projects.md]: ".PEM" -> ".pem"

File: User Guide/Projects.md [hunk 10]
```diff
@@ -746,6 +769,14 @@ How to **publish** via **HTTP Post**:
 **Step 4:** Set msg as **Body**
 
 ![Image not Available](/assets/Fig104.png)
+
+Here are the steps to publish a topic using HTTP REST in ADEX application:
+
+- Identify the endpoint: You need to know the endpoint URL of the server that is configured to receive the message. This URL will typically include the protocol (e.g., http:// or https://), the hostname or IP address of the server, and a path to the resource that will receive the message.
```
Comment (lohkianchai) [inline User Guide/Projects.md]: "(e.g., http:// or https://)" -> "(e.g. https://)"
"the hostname or IP address of the server" -> "the hostname of the server"

File: User Guide/Projects.md [hunk 11]
```diff
@@ -746,6 +769,14 @@ How to **publish** via **HTTP Post**:
...
+
+- Identify the endpoint: You need to know the endpoint URL of the server that is configured to receive the message. This URL will typically include the protocol (e.g., http:// or https://), the hostname or IP address of the server, and a path to the resource that will receive the message.	
+- Define the message payload: You need to define the message payload that will be sent in the HTTP POST request. The payload will typically be a JSON or XML object that contains the message data as well as any additional metadata that is required.
+- Set the HTTP headers: You need to set the HTTP headers for the request, including the Content-Type header that indicates the format of the message payload (e.g., application/json).
+- Send the HTTP POST request to send the HTTP POST request to the server.
+- If ADEX topic message payload is a JSON object that contains the topic name ("my/topic") and the message content ("Hello, world!"). The HTTP POST request is sent to the "ADEX Portal" endpoint with the Content-Type header set to "application/json". Once the request is sent, the ADEX application logs a success message if the request is successful, or an error message if there is an error.
```
Comment (lohkianchai) [inline User Guide/Projects.md]: 'message content ("Hello, world!")' -> 'message content {"Hello": "World!"}'
"the ADEX application logs a success" -> "the system logs a success"

File: User Guide/Projects.md [hunk 12]
```diff
@@ -835,7 +866,7 @@ You can refer for more details on mosquitto commands in on clicking the **[Link]
 
 ## My topics ##
 
-After creating a project, create your first topic. The **My topics** tab is only available for **Publisher**. This tab shows the topic that you have published for this project.
+After creating a project, you can create first topic. The **My topics** tab is only available for **Publisher**. This tab shows the topic that you have published for this project.
```
Comment (lohkianchai) [inline User Guide/Projects.md]: "you can create first topic." -> "you can create your first topic."

File: User Guide/Projects.md [hunk 13]
```diff
@@ -858,20 +889,20 @@ To **Create topic**:
...
 ![Image not Available](/assets/Fig46.png)
 
-**Step 6:**	Fill in the following details of the topic.
+**Step 6:** Fill in the following details of the topic.
 
 - **Topic description** – Enter description to the topic
 - **Tags** – Add tags to the topic maximum of 5 tags
 - **Is approval needed for subscription requests?**
-  - **Yes** – Approval is needed from publisher before you subscribe 
+  - **Yes** – Approval is needed from publisher before you subscribe to MQTT Client
```
Comment (lohkianchai) [inline User Guide/Projects.md]: "before you subscribe to MQTT Client" -> "before you can receive data to your MQTT Client"

File: User Guide/Projects.md [hunk 14]
```diff
@@ -884,21 +915,23 @@ You can see the newly created topic being added in **Projects** page > click **P
 
 ![Image not Available](/assets/Fig49.png)
 
-You can add more topics within the topic on your own project by selecting from topic folders and clicking the ![Image not Available](/assets/icon9.png) icon.
+You can add more topics within the topic on their own project by selecting from topic folders and clicking the ![Image not Available](/assets/icon9.png) icon.
```
Comment (lohkianchai) [inline User Guide/Projects.md]: "You can add more topics within the topic on their own project by selecting from topic folders" ->  "You can add more topics under a topic by selecting from the topic folder"

File: User Guide/Projects.md (hunk 1 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: ok KC updated.

File: User Guide/Projects.md (hunk 2 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: ok KC, done

File: User Guide/Projects.md (hunk 3 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: done

File: User Guide/Projects.md (hunk 4 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: done

File: User Guide/Projects.md (hunk 5 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: done

File: User Guide/Projects.md (hunk 6 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: done

File: User Guide/Projects.md (hunk 7 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: done

File: User Guide/Projects.md (hunk 8 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: done

File: User Guide/Projects.md (hunk 9 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: done

File: User Guide/Projects.md (hunk 10 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: done

File: User Guide/Projects.md (hunk 11 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: done

File: User Guide/Projects.md (hunk 12 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: done

File: User Guide/Projects.md (hunk 14 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: done

**Comment by venkanavaneetham** [comment]
KC, updated projects page based on ur comments, thanks.

File: User Guide/Projects.md (hunk 11 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: Removed "ADEX Portal"

**Review by lohkianchai — APPROVED** [review]
ok

**Pull request merged**